Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -66,6 +68,14 @@ public class PushDeploymentController : ApiController
         {
             using (_tracer.Step("ZipPushDeploy"))
             {
+                string deploymentId = null;
+                IEnumerable<string> idValues;
+
+                if (Request.Headers.TryGetValues(Constants.ScmDeploymentIdHeader, out idValues) && idValues.Count() > 0)
+                {
+                    deploymentId = idValues.ElementAt(0);
+                }
+
```

Comment:
Would be good to have a helper method for doing this like the StatusCode400() method in this class.

Reply:
Agreed.

Reply:
Taken care of.